{{cookiecutter.repo_name}}/Makefile

```diff
@@ -85,3 +66,29 @@ test-all: ## run tests on every supported Python/Django combination
 	tox
 
 validate: quality test ## run tests and quality checks
+
+## Localization targets
+
+extract_translations: ## extract strings to be translated, outputting .mo files
+	rm -rf docs/_build
+	cd {{cookiecutter.repo_name}} && ../manage.py makemessages -l en -v1 -d django
```

Comment:
i18n-tools can work with two different structures (conf/locale or locale). Note that this is the cookiecutter for Django apps, so we know that this will be in the correct structure.